src/child/containers/sidebars/favourites/Favourites.js

```diff
@@ -66,6 +68,24 @@ class Favourites extends Component {
         }
     }
 
+    onMouseOver(stockCode) {
```

Comment:
If we are to go with this approach, then I think this suggestion is nicer 👍 , however...

I'm not particularly keen on this approach for a few reasons:
+ It's imperative and manipulates the DOM directly (please correct me if I'm wrong and there's some React magic happening!)
  + Feels like a declarative approach would be nicer and more inline with the React way of thinking - perhaps using a prop could be an option (e.g. `isHovered` could be a prop used by the `Favourite` component to set the correct style, rather than adding a class directly)
+ It adds too much coupling (IMO) between the `Favourite` component and the `Favourites` container
  + The `Favourites` container has knowledge of the child `Favourite` component's DOM structure and directly manipulates it - this seems particularly brittle
  + The `Favourite` component relies on a parent component to ensure it's hover style works correctly - seems a bit excessive for simple UI state which could be handled with CSS if there weren't the DnD issues 

Couple of alternative ideas:
+ Make use of `props` (e.g. something like the `isHovered` suggestion above)
+ Make use of `state` locally within the `Favourite` component - I think this seems a bit more appropriate in this case, as it's only simple UI state and it removes the coupling; although, obvious downside is that being stateless would be simpler than stateful